symex-interop.el

```diff
@@ -93,7 +103,10 @@ right symex when we enter Symex mode."
          (rigpa-enter-lower-level))
         ((symex--evil-enabled-p)
          (evil-insert-state))
-        (t (evil-emacs-state))))
+        ((symex--evil-installed-p)
+         (evil-emacs-state))
+        ((fboundp 'symex-user-defined-lower-mode)
```

Comment:
Same here, we don't need these. When the `symex-meow` module is added, we can add an extra case at each of these sites.